.travis.yml

```diff
@@ -4,6 +4,9 @@ sudo: false
 rvm:
   - 2.1.0
 
+env:
+  - SKIP_CHECKS=yes
```

Comment:
why not a simple boolean here? then you could skip the match to "yes" in checks.rb